clients/runtime/src/error.rs

```diff
@@ -185,6 +215,22 @@ impl Error {
 	}
 }
 
+impl RecoverableError {
+	// The string which is used to match the error type
+	fn as_str(&self) -> &'static str {
+		match self {
```

Comment:
please add comment to know where these strings came from: 
```rust
   // based on https://github.com/paritytech/polkadot-sdk/blob/b0b4451f31dcd9da04e2952d7c4cdcc219271b88/substrate/primitives/runtime/src/transaction_validity.rs#L100-L118
```

Reply:
Both error strings are defined [here](https://github.com/pendulum-chain/spacewalk/pull/453/files#diff-6469b7c5aed83b33ec6bbd2a61154190361c333995a98a3c172436491a901b9bR207)